server-src/src/main/java/com/itt/kmt/models/User.java

```diff
@@ -0,0 +1,54 @@
+package com.itt.kmt.models;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import org.springframework.data.annotation.Id;
+
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * This class represents a User.
+ * 
+ * @author Rakshit Rajeev
+ */
+@Data
+@Getter
+@Setter
+public class User implements Serializable {
+    /**
+     * unique identifier.
+     */
+    @Id
+    private String id;
+    /**
+     * roleType of the user.
+     */
+    private String  userRole;
```

Comment:
This should be a reference of role object.

Reply:
Hi Shiva,
 we are going with String as per the schema.
{
   "userRole" : user
}